docs/TP6.md

```diff
@@ -0,0 +1,34 @@
+# TP6 - Server Side Rendering
+
+Pour utilser le server side rendering, nous allons migrer notre projet sur nuxt.js
+
```

Comment:
we could add link to nuxt website here :)

Reply:
Honestly, I've hesitated to add it during the writing. But if I do so, I must also add links for vue, vue-router and vuex...but i guess that the better